### ISSUE TITLE
Strong battery drain when speeding up audio

### ISSUE THREAD

**Issue by reidma** [TortugaPower/BookPlayer#763]
When listening at speeds higher than 1.0x with the screen off, my phone (iPhone XS Max with iOS 15.1) becomes hot and the battery drains at a rate around 1% per minute. I was listening at 1.3x when I noticed this happening. Switching to other speeds didn't seem to resolve the issue. I've been using the app for years on this phone and have never had this problem before. Closing all other apps, rebooting, and using low-power mode in iOS all had no effect.

**Comment by GianniCarlo** [comment]
Hmmm this may be due to extra calculations to get the adjusted remaining time when it's a different speed, I'll look into this for the next version

**Comment by reidma** [comment]
I did a bit more testing on this. I updated to iOS 15.3.1 and the issue still occurs. It also happens to some degree even at the default playback speed (1x). For comparison, I tried playing audio via several other apps (YouTube Music, Safari). None of them caused the sudden heating or battery drain. Battery drain using BookPlayer was 6-7% every ten minutes. In every case, using BookPlayer because my phone to heat up in my pocket in a very noticeable way, whereas no other audio app caused the same issue.
